tests/tests.rs

```diff
@@ -987,6 +987,57 @@ fn assert_exec_output(exec_style: &str) {
     }
 }
 
+/// Shell script execution using -exec
+#[test]
+fn test_exec_batch() {
+    assert_exec_batch_output("--exec-batch");
+}
+
+// Shell script execution using -x
```

Comment:
I think this is redundant. If you want to keep it, please modify the docstring to use `-X` instead of `-x`.